python/rmm/rmm/_cuda/stream.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import warnings
+
+from rmm.pylibrmm.stream import (
+    DEFAULT_STREAM,
+    LEGACY_DEFAULT_STREAM,
+    PER_THREAD_DEFAULT_STREAM,
+    Stream,
+)
+
+__all__ = [
+    DEFAULT_STREAM,
+    LEGACY_DEFAULT_STREAM,
+    PER_THREAD_DEFAULT_STREAM,
+    Stream,
```

Comment:
`__all__` should contain strings, not objects. https://docs.python.org/3/tutorial/modules.html#importing-from-a-package
```suggestion
    "DEFAULT_STREAM",
    "LEGACY_DEFAULT_STREAM",
    "PER_THREAD_DEFAULT_STREAM",
    "Stream",
```

Reply:
Thanks!